src/devtoolsPanel.ts

```diff
@@ -365,6 +365,36 @@ export class DevToolsPanel {
             `;
     }
 
+    private getCDNHtmlForWebview() {
```

Comment:
Could we rename this and every other instance of `CDN` to `Cdn` (e.g. `getCdnHtmlForWebview`)? I've messed up these capitalized abbreviations in var/func names enough to know this will come back to haunt me later.

See https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions?redirectedfrom=MSDN